.pochi/workflows/create-pr.md

```diff
@@ -1,7 +1,8 @@
 Please help create a PR for the current staging changes, following these guidelines:
+- If an issue number is not provided, use askFollowupQuestion to ask for it. If user provides a valid issue number, link to the issue with `Resolves <ISSUE_NUMBER>` at the beginning of the PR description.
```

Comment:
give an option of askFollowUpQuestion that help user find the issue. (have issue / no issue / find issue for me)

Reply:
you can optimize prompt to make it generates better output and confirm it with user.